zone/aa.cpp

```diff
@@ -933,7 +989,19 @@ void Client::SendAlternateAdvancementPoints() {
 	AATable_Struct* aa2 = (AATable_Struct *)outapp->pBuffer;
 
 	int i = 0;
+	if (RuleI(Character, StatCap) > 0)
+	{
+		aa2->aa_list[i].AA = detail::PhantomStatId;
+		aa2->aa_list[i].value = 1;
+		aa2->aa_list[i].charges = 0;
+		i++;
+	}
+
 	for(auto &aa : zone->aa_abilities) {
+		if (i >= 300) {
```

Comment:
Any reason we aren't just using `MAX_PP_AA_ARRAY` here

Reply:
Agreed